jsonpointer.js

```diff
@@ -1,5 +1,3 @@
-var console = require("console");
```

Comment:
I don't think that this module was build for the browser (the name contains node :smile:), so we can remove it.